podcastparser.py

```diff
@@ -686,6 +686,8 @@ def set_base(self, base):
         self.base = base
 
     def set_podcast_attr(self, key, value):
+        if key == 'type' and value not in ['episodic', 'serial']:
```

Comment:
Instead of doing this check here, it might also have been possible (cleaner?) to subclass `PodcastAttr` (e.g. `PodcastAttrOneOf('type', squash_whitespace, ('episodic', 'serial'))`) and deal with the special case there instead of here. I mean this definition above:

        'rss/channel/itunes:type': PodcastAttr('type', squash_whitespace),

Reply:
I took a slightly different approach that seemed OK -- let me know if you want me to go the full `PodcastAttrOneOf`